chain/x/oracle/client/cli/tx.go

```diff
@@ -122,10 +122,10 @@ $ %s tx oracle request 1 --calldata 1234abcdef --requested-validator-count 4 --s
 	}
 
 	cmd.Flags().BytesHexP(flagCalldata, "c", nil, "Calldata used in calling the oracle script")
-	cmd.Flags().Int64P(flagRequestedValidatorCount, "r", 0, "Number of top validators that need to report data for this request")
-	cmd.MarkFlagRequired(flagRequestedValidatorCount)
-	cmd.Flags().Int64P(flagSufficientValidatorCount, "v", 0, "Minimum number of reports sufficient to conclude the request's result")
-	cmd.MarkFlagRequired(flagSufficientValidatorCount)
+	cmd.Flags().Int64P(flagAskCount, "r", 0, "Number of top validators that need to report data for this request")
```

Comment:
Need to change short name of this field (r doesn't make sense anymore)

Reply:
Removed :P